runner/dl_checkpoints.sh

```diff
@@ -79,7 +79,42 @@ function download_all_models() {
 
     # Download live-video-to-video models.
     huggingface-cli download KBlueLeaf/kohaku-v2.1 --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
-    huggingface-cli download KwaiVGI/LivePortrait --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download stabilityai/sd-turbo --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download warmshao/FasterLivePortrait --local-dir models/FasterLivePortrait--checkpoints
```

Comment:
```suggestion
    huggingface-cli download warmshao/FasterLivePortrait --local-dir models/FasterLivePortrait--checkpoints
    huggingface-cli download yuvraj108c/Depth-Anything-Onnx --include depth_anything_vitl14.onnx --local-dir models
```
Would you mind adding also Depth Anything here? I had it in README, but this script is a better place I guess.

Reply:
I tried taking a look on the readme instructions but I think they will need some rethinking on the way the comfyui nodes are setup. I don't think we should use `--local-dir models` for example as that will download files to the root of the cache directory which would get messy. This likely needs its own sub directory as well similar to the faster live portrait and StreamDiffusion folders, but then the compilation script and docker image need to be updated too. If that ok I'll leave that refactor to a follow up PR for you, which has context on the comfyui image. Happy to review that

Reply:
Ok, let's leave it as a separate PR.